docs/usage/getting-started/use-cases.md

```diff
@@ -0,0 +1,135 @@
+# Use Cases
+
+This page describes some common use cases for Renovate, for those who are new and would like to get their heads around the capabilities.
+
+## Development Dependency Updates
```

Comment:
We Sure Use A Lot Of Capitalization In Our Headers.

Are we sure we need all those capitals, or can we get rid of some of them? 😄

Reply:
I don't really mind, so if you prefer less then feel free to propose more changes

Reply:
```suggestion
## Development dependency updates
```